bucket/usbimager.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.0.9",
+    "description": "A very minimal GUI app that can write compressed disk images to USB drives.",
+    "homepage": "https://bztsrc.gitlab.io/usbimager/",
+    "license": "MIT",
+    "url": "https://gitlab.com/bztsrc/usbimager/raw/binaries/usbimager_1.0.9-i686-win-gdi.zip",
+    "hash": "709120c6234fc9bd797eb1dd57f477909752bbe60dd8e084cbcefe727723568f",
+    "extract_dir": "USBImager",
+    "bin": "usbimager.exe",
```

Comment:
```suggestion
```
USBImager is not a CLI application.

Reply:
I wasn't completely sure regarding "bin" because of:

> If the program file is a GUI application and it doesn't accept any commandline arguments, no need to add it in bin. 
\* [CONTRIBUTING.md](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md)

Technically USBImager accepts arguments but I'd guess that they're not used that often: https://gitlab.com/bztsrc/usbimager#advanced-functionalities

Apart from that, thank you for reviewing and merging! =)